### PR TITLE
Add support for re-usable MeetingManager

### PR DESCRIPTION
**Pull request by devalevenkatesh**
**Issue #:** 
None.

**Description of changes:**
- Update `MeetingProvider` and corresponding documentation to support
  re-usable `MeetingManager` instance.

**Testing**
1. Have you successfully run `npm run build:release` locally? Yes.

2. How did you test these changes? Tested running the demo app locally.

3. If you made changes to the component library, have you provided corresponding documentation changes? Yes.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/providers/MeetingProvider/docs/MeetingManager.stories.mdx [hunk 1]
```diff
@@ -108,3 +112,43 @@ const MyApp = () => {
   }, [])
 }
 ```
+
+2. Share the same `MeetingManager` instance with multiple different `MeetingProvider`s.
+
+You can create a new `MeetingManager` instance and then pass it as a prop to your `MeetingProvider`s. If not passed,
+a new `MeetingManager` instance will be created internally with each `MeetingProvider` when rendered; and, 
+you will get the `MeetingManager` instance associated with that particular `MeetingProvider` when `useMeetingManager` hook is called.
```
Comment (p-foucht) [inline src/providers/MeetingProvider/docs/MeetingManager.stories.mdx:120]: We should very explicitly call out that this is to support unique cases where you may have different React trees within your applications (such as when migrating from an Angular application to React)

File: src/providers/MeetingProvider/docs/MeetingProvider.stories.mdx [hunk 2]
```diff
@@ -30,6 +30,41 @@ const App = () => (
...
+  useEffect(() => {
+    meetingManager.getAttendee = async (chimeAttendeeId: string, externalUserId?: string) => {
+      const response = await fetch('/my-attendees-endpoint);
+      const user = await response.json();
+
+      return {
+        name: user.name
+      }
+    }
+  }, [])
```
Comment (p-foucht) [inline src/providers/MeetingProvider/docs/MeetingProvider.stories.mdx:64]: I would remove this as it's not specific to the example of using two meeting managers

File: src/providers/MeetingProvider/docs/MeetingManager.stories.mdx (hunk 1 above)
Comment (devalevenkatesh) [reply src/providers/MeetingProvider/docs/MeetingManager.stories.mdx:122]: Yup, thanks, good point. I have added this in at the start of the `MeetingManager` documentation.

File: src/providers/MeetingProvider/docs/MeetingProvider.stories.mdx (hunk 2 above)
Comment (devalevenkatesh) [reply src/providers/MeetingProvider/docs/MeetingProvider.stories.mdx]: Cool, removed it. Thanks.

**Review by p-foucht — APPROVED** [review]

File: src/providers/MeetingProvider/docs/MeetingManager.stories.mdx [hunk 3]
```diff
@@ -4,6 +4,11 @@
 
 The `MeetingManager` is a class that helps integration with the SDK. It is tied to the `MeetingProvider` and is primarily responsible for joining, starting, and leaving your meeting.
 
+You can pass in the same `MeetingManager` instance to multiple different `MeetingProvider`s through props. This will override new `MeetingManager` instance creation inside
+the `MeetingProvider`. This is to support very specific cases where you may have different React trees within you applications. For example, you are migrating
+from Angular application to React.
```
Comment (p-foucht) [inline src/providers/MeetingProvider/docs/MeetingManager.stories.mdx:9]: from an Angular application*

Comment (devalevenkatesh) [reply src/providers/MeetingProvider/docs/MeetingManager.stories.mdx]: Updated, thanks!

**Review by p-foucht — APPROVED** [review]

**Pull request merged**